### PR TITLE
Bump to Nightwatch v3

### PR DESCRIPTION
**Pull request by AutomatedTester**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-nightwatch",
-  "version": "1.5.3",
+  "version": "4",
```
Comment (swrdfish) [inline package.json:3]: why 4?

File: src/init.ts [hunk 2]
```diff
@@ -610,26 +610,6 @@ export class NightwatchInit {
...
-            stdio: ['inherit', 'pipe', 'inherit'],
-            cwd: this.rootDir
-          });
-          Logger.info(colors.green('Done!'), '\n');
-        } catch (err) {
-          Logger.error(`Failed to install ${webdriver}. Please run 'npm install ${webdriver} --save-dev' later.\n`);
-        }
-      }
-    }
-
```
Comment (swrdfish) [inline src/init.ts:632]: Should we remove this part too: 
https://github.com/nightwatchjs/create-nightwatch/pull/93/files#diff-0e586e6c3496396d18792bb5dfc86ecfb5dd0fd43f81f33e55672bb2bee2ae79L973-L979

File: package.json (hunk 1 above)
Comment (AutomatedTester) [reply package.json]: I don't know... I had it 3 and then it was suddenly 4. fixed it

File: src/init.ts (hunk 2 above)
Comment (garg3133) [reply src/init.ts:632]: If Edge is automatically downloaded using selenium-manager, then sure!

Comment (garg3133) [reply src/init.ts:632]: Also, we should modify the `identifyWebdriverToInstall` method as well to not add the webdrivers in the `webdriversToInstall` array.

Comment (swrdfish) [reply src/init.ts:632]: I tested on my system, Edge driver seems to work fine with selenium manager.

**Pull request closed without merging**